pkg/api/wallet.go

```diff
@@ -47,3 +54,101 @@ func (s *Service) walletHandler(w http.ResponseWriter, r *http.Request) {
 		WalletAddress:             s.ethereumAddress,
 	})
 }
+
+type walletTxResponse struct {
+	TransactionHash common.Hash `json:"transactionHash"`
+}
+
+func (s *Service) walletWithdrawHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
the new endpoint should be unit tested
the new integration tested is much appreciated, btw

Reply:
added